src/omv/boards/ARDUINO_GIGA/omv_boardconfig.h

```diff
@@ -40,6 +40,8 @@
 
 #define OMV_HM01B0_ENABLE                   (1)
 #define OMV_HM0360_ENABLE                   (1)
+#define OMV_HM0360_XCLK_FREQ                (24000000)
+#define OMV_HM0360_PLL1_CONFIG              (0x04)
```

Comment:
This is half what it was before because the internal OSC clock is 48MHz. Then, it was divided by 4. So, now we use a 24 MHz clock and divide it by 2.